modules/nf-core/mindagap/main.nf

```diff
@@ -0,0 +1,32 @@
+process MINDAGAP {
+    tag "$meta.id"
+    label 'process_low'
+
+    container "rguerr/mindagap:latest"
```

Comment:
no biocontainer or conda?

Reply:
No unfortunately not. It's currently a more of a short library of scripts to process microscopy images from Resolve Bioscience's Molecular Cartogrpahy. I can ask if he wants to put it on bioconda for the future!

Reply:
we need a release for this for sure and adding it to bioconda would be great, too. I see there is already an issue for the versioning at least https://github.com/ViriatoII/MindaGap/issues/7

Reply:
Opened an issue regarding bioconda submission as well: https://github.com/ViriatoII/MindaGap/issues/8